cmd/loki/Dockerfile.cross

```diff
@@ -13,9 +13,21 @@ WORKDIR /src/loki
 RUN make clean && GOARCH=$(cat /goarch) GOARM=$(cat /goarm) make BUILD_IN_CONTAINER=false loki
 
 FROM alpine:3.9
-RUN apk add --update --no-cache ca-certificates
```

Comment:
I think this should remain earlier in the file for caching benefits.

Reply:
Moved back to the top